plugin/pkg/admission/webhook/config.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package webhook checks a webhook for configured operation admission
+package webhook
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/apiserver/pkg/admission"
+)
+
+const (
+	defaultDuration           = time.Duration(0)
+	defaultFailAction         = Deny
+	defaultRetryBackoff       = time.Duration(500) * time.Millisecond
+	errInvalidFailAction      = "webhook rule (%d) has an invalid fail action: %s should be either %v or %v"
```

Comment:
I can get behind that, but the reason it's like this is because the error is used in multiple places so either you copy/paste the text or you reuse it like this, or with a `make*` function.  I'm open to change it but that was my thinking.